test/migration.test.js

```diff
@@ -381,7 +381,7 @@ function setup(done) {
 }
 
 var query = function (sql, cb) {
-  db.adapter.query(sql, cb);
+  db.adapter.execute(sql, cb);
```

Comment:
For consistency. It's in the direction to remove usage of aliases to favor the official method (`execute` over `query`).